### PR TITLE
added release process documentation

### PR DESCRIPTION
**Pull request by lynxlynxlynx**
#242

### CODE REVIEW

File: Release-checklist.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+* visual inspection of laptop screens
+
+# Release
+* update changes.md
+  * update man page if necessary
+* bump version
+* github release + tags
+
+## Build releases for distribution
+###  Windows release on Linux:
```
Comment (selaux) [inline Release-checklist.md]: `make build-releases` can be used to build win release and Debian packages in one step

Comment (lynxlynxlynx) [inline Release-checklist.md]: This was just moved from the Makefile. It should eventually mention all the release targets (BSD notably missing).

**Comment by lynxlynxlynx** [comment]
I've added the missing target, so the only thing from #242 is deciding the way to document contributors.

**Comment by derSteFfi** [comment]
didn't we already agree on the shortlog for this?

**Comment by lynxlynxlynx** [comment]
I wasn't sure, so I didn't go ahead. Check the discussion ...

**Comment by derSteFfi** [comment]
well, I am pro shortlog. afair, no one had a problem with that and it's the easiest solution for a release.

**Comment by selaux** [comment]
What about adding the checklist here. I found it useful for this release.

**Comment by lynxlynxlynx** [comment]
to reduce duplication, just convert the existing thing to a checklist.

**Pull request merged**